src/SingleArticleCard.jsx

```diff
@@ -0,0 +1,17 @@
+export const SingleArticleCard = ({ article }) => {
+	console.log(article, '<<< ARTICLE CARD');
+	console.log(article.article.article_id);
+	return (
+		<li className='card'>
```

Comment:
Where is the value in returning this as a list item when we know it's only a single article? Where is the article body? This should be the full article for a user to read.